modules/core/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@nebula.gl/core",
+  "description": "A suite of 3D-enabled data editing overlays, suitable for deck.gl",
+  "license": "MIT",
+  "version": "0.12.1",
+  "author": "Georgios Karnas <georgios@uber.com>",
```

Comment:
Suggestion: Change to "contributors" so that more names can be listed?

Reply:
Agreed. We can do that in a separate PR